src/amo/containers/App.js

```diff
@@ -20,9 +34,22 @@ export class AppBase extends React.Component {
           defaultTitle={i18n.gettext('Add-ons for Firefox')}
         />
         {children}
+        <footer>{this.logIn()}</footer>
       </div>
     );
   }
 }
 
-export default translate({ withRef: true })(AppBase);
+export const makeMapStateToProps = ({ _window } = {}) => (state) => ({
+  isAuthenticated: !!state.auth.token,
+  handleLogIn() {
+    // eslint-disable-next-line no-param-reassign
+    (_window || window).location = startLoginUrl();
```

Comment:
It shouldn't get called on the server, that being said I'm trying to sort out the test failures and some of this is changing. I'm moving this into the component so it's more clear that it will only happen on the client.